coalib/settings/ConfigurationGathering.py

```diff
@@ -118,10 +119,33 @@ def warn_nonexistent_targets(targets, sections, log_printer):
     # functions have intended side effects though.
     files_config_absent = warn_config_absent(sections, 'files', log_printer)
     bears_config_absent = warn_config_absent(sections, 'bears', log_printer)
+    local_bears, global_bears = collect_all_bears_from_sections(
```

Comment:
This seems redundant (and maybe slow things down) because we already collect list of bears from here https://github.com/coala/coala/blob/master/coalib/settings/ConfigurationGathering.py#L402.

IMO its better to separate the function (`warn_no_bears_found` or something) because this is not really related to `warn_nonexistent_targets`. And then pass the list of bears from `gather_configuration`.